MAUI/UIKit/GettingStarted.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Getting Started with MAUI UI Kit control | Syncfusion
```

Comment:
Check Title and Description all MD files

Reply:
Checked and Updated in Latest Commit